dist-persist/wbstack/src/loadInternal.php

```diff
@@ -24,6 +25,7 @@
 $wgAPIModules['wbstackElasticSearchInit'] = \WBStack\Internal\ApiWbStackElasticSearchInit::class;
 $wgAPIModules['wbstackForceSearchIndex'] = \WBStack\Internal\ApiWbStackForceSearchIndex::class;
 $wgAPIModules['wbstackQueueSearchIndexBatches'] = \WBStack\Internal\ApiWbStackQueueSearchIndexBatches::class;
+$wgAPIModules['wbstacksitestatsupdate'] = \WBStack\Internal\ApiWbStackSiteStatsUpdate::class;
```

Comment:
```suggestion
$wgAPIModules['wbstackSiteStatsUpdate'] = \WBStack\Internal\ApiWbStackSiteStatsUpdate::class;
```
Nothing dramatic but I see the other module names are written camelCase, could we use it here too?

Reply:
Yeah sure will probably need to change that on the api too

Reply:
these https://github.com/wbstack/api/pull/499/files#diff-5a8ae3065e9b4af4cbba73b1b3b35752a60680bd5d0372f8ebee122ab8581857R70